### DISCUSSION TITLE
Many nodes going to a single box

### DISCUSSION THREAD

**Discussion by josephernest** [tone-row/flowchart-fun#26] [Features / Syntax]
Hi, congrats for your tool!

In order to do something like "many nodes  go to 1 node":

![image](https://user-images.githubusercontent.com/6168083/109792948-590ac400-7c14-11eb-8ee5-c8cc868331c9.png)

is there a better way than doing:

    A
      container
    B
      (2)
    C
      (2)
    D
      (2)
    E
      (2)

to avoid the repetition of newlines with `(2)`?

**Comment by rob-gordon** [comment]
An alternative (though not more concise) is using an [id]. I just wanted to mention that because then if your lines shift you won't need to rewrite the line numbers.
```
A
 [container] container
B
 (container)
C
 (container)
D
 (container)
```

There is not, at present, any shorter way to write. I guess the syntax really favors more children over more parents (tree shapes) 😅

I don't want to add anything to the syntax without gathering more feedback so I'm going turn this issue into a discussion where people can show interest / suggest a syntax to accomplish this.

**Reply by josephernest** [reply]
> I guess the syntax really favors more children over more parents (tree shapes)

@rob-gordon Yes exactly, that's the key thing on this topic ! Your syntax is great as it currently is, so I totally agree that you should wait for feedback before modifying anything.

More generally, I am curious if there is a way to have a syntax which is both simple (like yours), and able to nicely represent *directed graphs* (not only tree shapes).

Sidenote: I have looked at the other tools that people mentioned in your ShowHN discussion, and most of these tools are much more complex and less intuitive than yours. So you clearly win on this "simplicity" aspect which is great.

Being able to handle complex graphs (non tree-shaped) and keep the simplicity aspect would be amazing!
